static/js/subs.js

```diff
@@ -552,7 +552,6 @@ function ajaxSubscribe(stream) {
         data: {"subscriptions": JSON.stringify([{"name": stream}]) },
         success: function (resp, statusText, xhr, form) {
             $("#create_stream_name").val("");
-
```

Comment:
oh, sorry that wasn't clear, I just meant that this commit shouldn't be deleting this unrelated blank line from the code, and I'd like you to rebase and restore the blank line.  A "hunk" is a just a technical word for a section of a diff/patch.